### PR TITLE
KOIB-11-Install-and-setup-MUI-and-styled-component

### PR DESCRIPTION
**Pull request by susan8848**
>Installed MUI and styled components
>Configured babel in .babelrc

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -1,95 +1,128 @@
-import Image from 'next/image';
-import styles from './page.module.css';
+// import Image from 'next/image';
```
Comment (dipeshkumarsah98) [inline app/page.tsx:1]: why are we commenting this whole code ?
if its not necessary, delete it

File: app/page.tsx [hunk 2]
```diff
@@ -1,95 +1,128 @@
...
+//           <p>
+//             Instantly deploy your Next.js site to a shareable URL with Vercel.
+//           </p>
+//         </a>
+//       </div>
+//     </main>
+//   );
+// }
+
+'use client';
```
Comment (dipeshkumarsah98) [inline app/page.tsx:97]: why to use 'use client' ?

File: app/page.tsx [hunk 3]
```diff
@@ -1,95 +1,128 @@
...
+
+import * as React from 'react';
```
Comment (dipeshkumarsah98) [inline app/page.tsx:99]: if there is sample file where you have installed and created sample of MUI. then why importing whole mui styles ?

**Review by dipeshkumarsah98 — CHANGES_REQUESTED** [review]
Nice work 👍 .  While reviewing your PR, I notice some changes are required which i have commented below. when you resolve all the comments, ask for re-review. Thank you.

**Comment by subhashishshah** [comment]
Good work! Description can be better. Install MUI doesn't make sense. Also explain, how we reviewers can test this PR.

**Comment by subhashishshah** [comment]
Also, the ticket has asked to install styled component. I haven't seen styled component package in this PR. Install MUI and styled component both and setup. If you have problem, don't hesitate to ask @dipeshkumarsah98 or me.

**Pull request merged**